src/postersBlock/postersBlock.component.js

```diff
@@ -0,0 +1,79 @@
+import React, { memo, useEffect, useState } from 'react';
+import { getPosters, getPostersByCategory } from '../servise/posterService';
+import { DateWrapper, Img, PosterInfoWrapper, PostersWrapper, PosterTitle, PosterWrapper } from './postersBlock.styled';
+import PosterCount from './postersCount/postersCount.component';
+import { MainWrapper } from '../global/globalStyles';
+import PosterCategory from './posterCategory/posterCategory.component';
+import NavigationCategory from './navigationCategory/navigationCategory.component';
+import PropTypes from 'prop-types';
+
+const links = [
+  { id: 0, text: 'all' },
+  { id: 1, text: 'documentary' },
+  { id: 2, text: 'comedy' },
+  { id: 3, text: 'horror' },
+  { id: 4, text: 'crime' },
+];
+const PostersBlock = () => {
+  const [posters, setPosters] = useState([]);
+  const [countPosters, setCountPosters] = useState(0);
+  const [activeNav, setActiveNav] = useState(links[0]);
```

Comment:
it is better to use function syntax for such default values
` useState(() => links[0])`